core/src/main/scala/uk/gov/nationalarchives/tdr/api/core/monitoring/Metrics.scala

```diff
@@ -0,0 +1,39 @@
+package uk.gov.nationalarchives.tdr.api.core.monitoring
+
+import software.amazon.awssdk.services.cloudwatch.CloudWatchAsyncClient
+import software.amazon.awssdk.services.cloudwatch.model.{Dimension, MetricDatum, PutMetricDataRequest}
+import uk.gov.nationalarchives.tdr.api.core.db.dao.ConsignmentDao
+
+import scala.concurrent.ExecutionContext
+
+class Metrics(consignmentDao: ConsignmentDao)(implicit val executionContext: ExecutionContext) {
+  private val metricNamespace = sys.env.getOrElse("CLOUDWATCH_METRIC_NAMESPACE", "TdrPrototypeDev")
+
+  def recordConsignmentCreation(transferringBody: String): Unit = {
+    recordConsignmentMetric("ConsignmentCreated", transferringBody)
+  }
+
+  def recordConsignmentConfirmation(consignmentId: Int): Unit = {
```

Comment:
Maybe rename this to: def recordConsignmentTransferConfirmation

Reply:
Done :+1: